addons/zotero/views.py

```diff
@@ -57,12 +61,19 @@ def library_list(self):
         def _library_list(auth, node_addon, **kwargs):
             """ Returns a list of group libraries - for use with Zotero addon
             """
+
             limit = request.args.get('limit')
             start = request.args.get('start')
             return_count = is_truthy(request.args.get('return_count', False))
             append_personal = is_truthy(request.args.get('append_personal', True))
-            return node_addon.get_folders(limit=limit, start=start, return_count=return_count, append_personal=append_personal)
+            if not waffle.flag_is_active(request, features.ENABLE_GV):
```

Comment:
nit: I'd also flip this statement

Reply:
Moreover, there is a change that we don't need lib library list anymore

Reply:
I tried to remove library_list but too many errors appeared